schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
-
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE
+    animals (
+        id SERIAL PRIMARY KEY,
```

Comment:
[OPTIONAL] I suggest you mark the id column as primary key. It is unique and you can use this as a foreign key on other related tables.